src/mqt/core/__init__.py

```diff
@@ -2,7 +2,24 @@
 
 from __future__ import annotations
 
-from ._core import add
+from ._core import (
+    CompoundOperation,
+    Control,
+    Expression,
+    OpType,
+    Permutation,
+    QuantumComputation,
+    StandardOperation,
+)
 from ._version import version as __version__
 
-__all__ = ["__version__", "add"]
+__all__ = [
+    "__version__",
+    "OpType",
+    "StandardOperation",
+    "CompoundOperation",
+    "Permutation",
+    "QuantumComputation",
+    "Control",
+    "Expression",
```

Comment:
Nitpick: the order here is not consistent with the includes above.

Reply:
I bet at some point there will be a ruff check for that 🙃

Reply:
Ordered it.